analysis/Scripts/snakefile

```diff
@@ -7,20 +7,35 @@ configfile: 'cfg.yaml'
 rule all:
      input:[ ]
 
-rule mass_fitting_bins:
+rule mass_fitting_bins_GaussCB:
```

Comment:
we should discuss whether to have multiple rules, or  just refit with both shapes every time (or have both options)

Reply:
(can use nested rules for example)